ci/azure-pipelines-nightly.yml

```diff
@@ -124,6 +124,28 @@ jobs:
     displayName: "Run the tests"
   - template: ../ci/publish-bazel-test-results.yml
 
+- job: cryptotest_cw310
+  displayName: "Cryptotest Framework CW310 Tests"
+  timeoutInMinutes: 300
+  dependsOn: checkout
+  pool:
+    name: FPGA
+    demands: BOARD -equals cw310
+  steps:
+  - template: ../ci/checkout-template.yml
+  - template: ../ci/install-package-dependencies.yml
+  - bash: |
+      set -x
+      ci/bazelisk.sh test \
+        --define DISABLE_VERILATOR_BUILD=true \
+        --define bitstream=gcp_splice \
+        --build_tests_only \
+        --test_output=errors \
```

Comment:
I wrote `--test-output=errors` in order to not burden the CI with piping the output from the cryptotest framework (which there is a lot of) to the console, but we will need a way to communicate which tests failed, if any. Currently this is printed to the test log, but I'm not sure if there's a way to recover that file from CI. This does not affect the random seed printed for random tests.

Reply:
Edit: outdated; getting the desired test summary to print required some more in-depth handling of logging in each of the test harnesses.